osu.Game/Screens/Edit/Compose/Components/Timeline/SamplePointPiece.cs

```diff
@@ -94,6 +96,9 @@ private void load()
                     }
                 };
 
+                bank.TabbableContentContainer = flow;
+                volume.TabbableContentContainer = flow;
```

Comment:
These cannot be inlined into the property initialiser hierarchy above because the compiler complains that `flow` is potentially unassigned at that point.